src/containers/settings/EditServiceScreen.js

```diff
@@ -198,6 +204,13 @@ class EditServiceScreen extends Component {
           value: service.isBadgeEnabled,
           default: DEFAULT_SERVICE_SETTINGS.isBadgeEnabled,
         },
+        trapLinkClicks: {
+          label: intl.formatMessage(messages.trapLinkClicks),
+          // THIS OVERRIDES THE CURRENT EXISTING RECIPES SETTINGS IF THE GLOBAL SETTING IS SET TO TRUE. 
+          // TODO: SOLUTION FOR WHEN GLOBAL SETTING IS TRUE AND THE USER WANTS TO SET A SPECIFIC SERVICE TO OFF.
+          value: isTrapLinkClicksDefault || service.trapLinkClicks,
```

Comment:
I dont think the Edit screen should be where this global and service-level are "logically merged". Please keep them separate.

Reply:
That override should happen in the place where the recipe is initialized (somewhere before this line is executed): https://github.com/vraravam/ferdium-app/blob/develop/src/webview/recipe.js#L215